lib/osf-components/addon/components/ancestry-display/template.hbs

```diff
@@ -0,0 +1,17 @@
+<span local-class='Titles'>
```

Comment:
Whenever you make a component with `@tagName('')`, it's good to add `...attributes` to its main element (in this case, this `span`). It makes the component a lot more flexible for the future.